docs/03.API-EXAMPLE.md

```diff
@@ -147,8 +503,8 @@ main (void)
   jerry_value_t global_object = jerry_get_global_object ();
 
   /* Constructing strings */
-  jerry_value_t prop_name = jerry_create_string ((const jerry_char_t *) "s");
-  jerry_value_t prop_value = jerry_create_string (str);
+  jerry_value_t prop_name = jerry_create_string ((const jerry_char_t *) "my_var");
+  jerry_value_t prop_value = jerry_create_string ((const jerry_char_t *) "Hello from C!");
 
   /* Setting the string value as a property of the Global object */
   jerry_release_value (jerry_set_property (global_object, prop_name, prop_value));
```

Comment:
`jerry_set_property` may returns with error, so I'd suggest to add the error check to the example in the same way, as you did above.